cmd/brew-test-bot.rb

```diff
@@ -977,7 +980,10 @@ def test_ci_upload(tap)
 
     if (pr = ENV["UPSTREAM_PULL_REQUEST"])
       pull_pr = "https://github.com/#{tap.user}/homebrew-#{tap.repo}/pull/#{pr}"
-      safe_system "brew", "pull", "--clean", pull_pr
+      args = []
+      args << "--tap=#{@tap}" unless @tap.nil?
+      args << pull_pr
+      test "brew", "pull", "--clean", *args
```

Comment:
My suggested solution would be to build bottles on Homebrew/homebrew-science if it's meant to support Linux (which I don't think is wise, as I've explained before, but I think that ship has probably sailed) or make Linuxbrew/linuxbrew-science a proper fork with the various Linux bottles in it.